README.md

```diff
@@ -149,13 +153,16 @@ In the playbook below, the parameters are initialised directly in the yaml file
 |`controller_account_name` |  Account name | All |
 |`controller_port`   | The controller port   | All |
 |`enable_ssl`   | Indicate if SSL is enabled in the controller or not | All |
+|`restart_app`| Set this flag `true` to restart services we are instrumenting (i.e. IIS or tomcat) | All
 |`agent_log_level` | set the log level for the agent. valid options are : **info, trace, debug, warn, error, fatal, all** and **off**. This setting is applied to all the loggers named in the **`agent_loggers`** list| Machine, DB, Java
 |`agent_loggers` | List of loggers to set the log level on. The logger names vary from agent to agent. The default is set to ['com.singularity','com']. Update this variable with loggers specific to the target agent as required (refer to the log4j files in the <get-home>/conf/logging directory for more info). | Machine, DB, Java
 |`db_agent_name` | Name assigned to the agent, typically used to allow one Database Agent  to act as a backup to another one | DB
 |`install_jre`| Set this parameter to false if the JRE should not be installed together with the DB agent. <br><br>**Note:** to install java on windows, you need to run the <i>install-roles.yml</i> playbook first, which adds a galaxy role (lean_delivery.java) to you local playbook folder | DB
-|`services`| List of stand-alone services to be instrumented with the .NET agent| .NET
 |`monitor_all_IIS_apps`| Enable automatic instrumentation of all IIS applications | .NET
 |`runtime_reinstrumentation` | Runtime re-instrumentation works for .NET Framework 4.5.2 and greater. Note: Make sure you test this first in a non-production environment | .NET |
+|`machine_agent` | YAML map that describes dotnet machine_agent settings. See roles/dotnet/defaults/main.yml for the example | .NET |
```

Comment:
I'd suggest we qualify this variable name to avoid it from confusing customers. Something along the line of `dotnet_agent_machine_settings`, `dotnet_agent_machine_config`.

Reply:
As a rule of the thumb, I would prefix with role name then: dotnet_

Reply:
Renamed to dotnet_machine_agent